test/ctia/test_helpers/core.clj

```diff
@@ -27,6 +27,21 @@
 
 (def ^:dynamic ^:private *config-transformers* [])
 
+(defn get-service-map [app svc-kw]
+  {:pre [(keyword? svc-kw)]}
+  ;; stub implementation until trapperkeeper bootstrap
+  (assert (= ::global-app app)
+          "app must be the result of get-current-app")
+  (case svc-kw
+    :ConfigService {:get-config p/get-global-properties
+                    :get-in-config p/get-in-global-properties}
+    :StoreService {:all-stores (fn [] @store/stores)
+                   :read-store store/read-store
+                   :write-store store/write-store}
+    (throw (ex-info (str "No service: " svc-kw)
+                    {:app app
+                     :service svc-kw}))))
```

Comment:
This has a general implementation for TK that I discovered how to write recently.

```clojure
(defn get-service-map [app svc-kw]
  {:pre [(keyword? svc-kw)]}
  (let [graph (app/service-graph app)
        m (svc-kw graph)]
    (assert (map? m) (str "No service " svc-kw ", found " (keys graph)))
    m))
```